app/web/components/LanguagePickerSelect.tsx

```diff
@@ -0,0 +1,195 @@
+import CheckIcon from "@mui/icons-material/Check";
+import ExpandMoreOutlinedIcon from "@mui/icons-material/ExpandMoreOutlined";
+import {
+  Box,
+  FormControl,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Select as MuiSelect,
+  SelectChangeEvent,
+  Stack,
+  TextField,
+  useMediaQuery,
+  useTheme,
+} from "@mui/material";
+import { LANGUAGE_MAP } from "i18n/constants";
+import { useState } from "react";
+
+type LanguagePickerSelectProps = {
+  defaultValue?: string;
+  value?: string;
+  onSelect?: (value: string) => void;
+  displayMode?: "round" | "rect";
+};
+
+// note: manually add a non-secure cookie in development to test this functionality
+// TODO: this function lives in two places... where should it live?
+function getLangCookie() {
+  const name = "couchers-preferred-language=";
+  const decodedCookie = decodeURIComponent(document.cookie);
+  const ca = decodedCookie.split(";");
+  for (let i = 0; i < ca.length; i++) {
+    let c = ca[i];
+    while (c.charAt(0) == " ") {
+      c = c.substring(1);
+    }
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length);
+    }
+  }
+  return "";
+}
+
+export default function LanguagePickerSelect({
+  defaultValue,
+  value,
+  onSelect,
+  displayMode = "round", // default to round if not specified
+}: LanguagePickerSelectProps) {
+  const [language, setLanguage] = useState(getLangCookie());
+  const theme = useTheme();
+  const isMdOrWider = useMediaQuery(theme.breakpoints.up("md"));
```

Comment:
We've been using `isMobile` as a variable name as it's a bit more clear. Would you mind switching this to that? You'll need to reverse the logic since this is now `isNotMobile`.